.github/workflows/main.yml

```diff
@@ -68,7 +68,7 @@ jobs:
         with:
           registry: ghcr.io
           username: ${{ github.repository_owner }}
```

Comment:
username: ${{ github.actor }}
          password: ${{ secrets.GITHUB_TOKEN }}

if you switch to a github actor you can use the github provided token